test/object-store/sync/flx_sync.cpp

```diff
@@ -1196,6 +1269,7 @@ TEST_CASE("flx: client reset", "[sync][flx][client reset][baas]") {
         auto subs = realm->get_latest_subscription_set();
         auto result = subs.get_state_change_notification(sync::SubscriptionSet::State::Complete).get();
         CHECK(result == sync::SubscriptionSet::State::Complete);
+        SyncSession::OnlyForTesting::pause_async(*realm->sync_session()).get();
```

Comment:
FYI - in case you weren't aware, the client reset triggered by the command below just invalidates the file ident and the reset is not initiated until the session reconnects.